examples/twig/README.md

```diff
@@ -0,0 +1,33 @@
+# Fractal Twig example
+
+This is an example instance of Fractal using the Twig adapter.
+
+## Setup
+
+[Follow these instructions](https://github.com/frctl/fractal#development) to setup the development environement, then run:
+
+```bash
+npm start
+```
+
+to start Fractal server in development mode, or:
+
+```
+npm run build
+```
+
+to build a static version in the `dist` folder.
+
+## Tests
```

Comment:
Now that tests are defined in the root package.json, this section should either be removed from this file (and Handlebars example) or rephrased to specify this must be run from the root directory.

Reply:
Yup, removed from both.